index.js

```diff
@@ -11,14 +11,21 @@ const nextButton = document.querySelector('[data-js="button-next"]');
 const pagination = document.querySelector('[data-js="pagination"]');
 
 // States
-const maxPage = 1;
-const page = 1;
-const searchQuery = "";
+let maxPage = 1;
+let page = 1;
+let searchQuery = "";
 
 async function fetchCharacters() {
   try {
-    const response = await fetch("https://rickandmortyapi.com/api/character");
+    const response = await fetch(
+      `https://rickandmortyapi.com/api/character?page=${page}&name=${encodeURIComponent(
+        searchQuery
+      )}`
+    );
+    console.log(response);
     const data = await response.json();
+    maxPage = data.info.pages;
+    console.log(maxPage);
     cardContainer.innerHTML = "";
     data.results.forEach((character) => {
       const cardElement = CharacterCard(character);
```

Comment:
does the ocurences api work like this? If not you might need a reference to the episode length

Reply:
we already implemented the episode.length as a reference to the object element. it works.